src/platform/packages/shared/response-ops/alerts-table/components/last_updated_at.tsx

```diff
@@ -75,7 +70,14 @@ export const LastUpdatedAt = React.memo<LastUpdatedAtProps>(
     return (
       <EuiToolTip
         content={<Updated date={date} prefix={prefix} updatedAt={updatedAt} />}
-        anchorProps={anchorStyles}
+        anchorProps={{
```

Comment:
This is to avoid layout shifts in the toolbar when only the custom controls are shown (labels are shorter than empty buttons, so when toggling bulk action controls the toolbar would change height)